server/src/main/java/org/apache/uniffle/server/ShuffleServerConf.java

```diff
@@ -430,7 +430,9 @@ public class ShuffleServerConf extends RssBaseConf {
           .withDescription(
               "The type for shuffle buffers. Setting as LINKED_LIST or SKIP_LIST."
                   + "The default value is LINKED_LIST. SKIP_LIST will help to improve"
-                  + "the performance, but it don't support the show start feature of MR");
+                  + "the performance especially when allocating a large amount of memory"
```

Comment:
This might be a bit vague. What does "large amount of memory" mean? To give an extreme example, if there are only a few ShuffleBuffers in the list, and these ShuffleBuffers are very large, does this also count as a large amount of memory? Or is it meant to solve the scenario where there are a large number of blocks in the list?

Also, you need to add a space before `the performance`.

Reply:
I can't describe so many details unless i do a lot of testing. The general users maybe don't understand in what scenarios will there be a lot of blocks. Users still need to test according to their own scenarios.

Reply:
Updated. Actually i think skip list will improve performance in most cases. Even if the blocks are not many. The sorting performance is not good when flush currently because it will create a reference copy from `blocks`.

Reply:
Oh yeah, this is another performance improvement, or maybe we can update it like:

SKIP_LIST will help to improve the performance when there are a large number of blocks in memory or when the memory occupied by the blocks is very large.

WDYT?